### PR TITLE
Vue export 2

### PR DESCRIPTION
**Pull request by blaisefonseca**

### CODE REVIEW

File: app/components/common/export.vue [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+                    {{$t('download')}}
+                </button>    
+            </div>
+        </modal>
+    </span>   
+</template>
+
+<script>
+    import { Modal } from 'vue-2-bootstrap-3'
+	import i18n from '../../locales/en/components/export.json';
```
Comment (randyhoulahan) [inline app/components/common/export.vue:79]: does export.json filename accurately represent the contents?

File: app/components/common/export.vue [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+                setTimeout(()=>{
+                    $('#datatable').tableExport({
+                        formats: ["xlsx", "xls", "csv"],
+                        filename: fileName,
+                    });
+                    $(`.${this.downloadFormat}`).click(); 
+
+                    this.downloadDocs = this.downloadDocs.slice(0, 24);
+                    this.loading = false;
+                }, 500);
```
Comment (randyhoulahan) [inline app/components/common/export.vue:138]: I know we should try not to use timeouts, but I cannot offer an alternative here either.

File: app/components/kb/filters.js [hunk 3]
```diff
@@ -10,4 +11,10 @@ export function formatDate(datetime, format) {
 export function formatMomentDate(datetime, method, arg1, arg2, arg3) {
     if(datetime)
         return moment.utc(datetime)[method](arg1, arg2, arg3);    
-}
\ No newline at end of file
+}
+
+
+Vue.filter('formatDate'         , formatDate);
```
Comment (randyhoulahan) [inline app/components/kb/filters.js:17]: My preference is to load where needed.  So in the using component, import then add to the vue component.  Assuming we will add more and more filters.

**Review by randyhoulahan — APPROVED** [review]
Just a couple comments nothing requiring change.

File: app/components/common/export.vue (hunk 1 above)
Comment (blaisefonseca) [reply app/components/common/export.vue:79]: yes

File: app/components/common/export.vue (hunk 2 above)
Comment (blaisefonseca) [reply app/components/common/export.vue:138]: could not fins alternative myself

**Pull request merged**